src/main/java/com/oneops/proxy/web/ErrorController.java

```diff
@@ -52,8 +53,27 @@ public ErrorController(ErrorAttributes errAttrs, ServerProperties serverProperti
   public @ResponseBody ResponseEntity<ErrorResponse> errorResponse(
       HttpServletRequest req, HttpServletResponse res) {
     Map<String, Object> body = getErrorAttributes(req, isIncludeStackTrace(req, MediaType.ALL));
-    HttpStatus status = getStatus(req);
 
+    /*---------------Update status code and format error respose for RequestRejectedException------------------*/
+    Object message = null;
+    Object path = null;
+    String exceptionName = body.get("exception").toString();
```

Comment:
Can it throw NPE any chance? ie, body.get("exception") retuning null.

Reply:
This will be called when any exceptions is coming. "exception" will get some value .